src/components/modal/stories/ModalDialog.stories.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import * as React from 'react';
+import { IntlProvider } from 'react-intl';
+
+import ModalDialog from '../ModalDialog';
+import notes from './ModalActions.stories.md';
```

Comment:
should this be `./ModalDialog.stories.md`?

Reply:
Yes, good catch!